PSPrintSDK/OLProductTemplateSyncRequest.m

```diff
@@ -77,6 +136,13 @@ - (void)fetchTemplatesWithURL:(NSURL *)url templateAccumulator:(NSMutableArray *
                                 
                                 if (costPerSheetByCurrencyCode.count > 0) {
                                     OLProductTemplate *t = [[OLProductTemplate alloc] initWithIdentifier:identifier name:name sheetQuantity:[imagesPerSheet unsignedIntegerValue] sheetCostsByCurrencyCode:costPerSheetByCurrencyCode enabled:enabled];
+                                    t.coverPhotoURL = [NSURL URLWithString:coverPhoto];
+                                    t.productPhotographyURLs = productShots;
+                                    t.templateClass = [OLProductTemplate templateClassWithIdentifier:productClass];
```

Comment:
OLProductTemplate templateClassWithIdentifier: will crash and burn horribly if we add new product classes that the SDK doesn't know about yet, which is conceivable as older versions start to exist. I'd future proof all this parsing by simply skipping over the product/template if it doesn't make sense.